clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -5430,6 +5430,9 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
     if (Arg *A = Args.getLastArg(options::OPT_fsycl_range_rounding_EQ))
       A->render(Args, CmdArgs);
 
+    if (Arg *A = Args.getLastArg(options::OPT_fsycl_exp_range_rounding))
+      A->render(Args, CmdArgs);
```

Comment:
Should there be a diagnostic when used with `-fsycl-range-rounding` that `-fsycl-exp-range-rounding` overrides (regardless of ordering)

Reply:
Perhaps `-fsycl-range-rounding=disable` and `-fsycl-exp-range-rounding` should give a compilation error when used together?

Reply:
The current behaviour is that disabling range rounding will override the exp range rounding. I actually think this is an OK default. Let me know what you think

Reply:
Yes, this is fine by me.  It would be good to let the user know that usage of `-fsycl-range-rounding=disable -fsycl-exp-range-rounding` results in disabling range rounding behaviors.  typical expectation on options that are similar is 'last one wins'

Reply:
OK I will add this to the documentation

Reply:
Added. Let me know what you think